pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <groupId>io.gravitee.maven.archetypes</groupId>
     <artifactId>gravitee-policy-maven-archetype</artifactId>
-    <version>1.9.0-SNAPSHOT</version>
+    <version>1.9.0</version>
```

Comment:
The version has to remain snapshot until released

Reply:
Fixed